### PR TITLE
commit ex3 - extension

### PR DESCRIPTION
**Pull request by IsraelCohen54**
commit ex3 - extension

### CODE REVIEW

File: src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.hibob.kotlin_exercises
+
+import kotlin.math.pow
+
+// q1:
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt:5]: you can delete the q it is for your use only

File: src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+//fun sum(list: List<Int>):Int {
+//    var result = 0
+//    for (i in list) {
+//        result += i
+//    }
+//    return result
+//}
+
+// Ans_1:
+fun List<Int>.sum() : Int{
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt:15]: good! you can map{} use in the future, look for it in google

File: src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    return sum
+}
+
+//import kotlin.math.pow
+infix fun Number.toPowerOf(exponent: Number): Double {
+    return this.toDouble().pow(exponent.toDouble())
+}
+
+fun main() {
```
Comment (liorsiv973) [inline src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt:28]: you can delete it in the pr it for your checks only

**Review by liorsiv973 — APPROVED** [review]
great work

File: src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt (hunk 2 above)
Comment (IsraelCohen54) [reply src/main/kotlin/com/hibob/kotlin_exercises/extensions_ex3.kt:15]: Thanks!|
Why does it matter if I do sum vs using map? (its not a for loop, isn't it?)

**Pull request merged**